apps/cookbook/src/app/showcase/modal-experimental-showcase/modal-experimental-showcase.component.ts

```diff
@@ -0,0 +1,270 @@
+      signature: 'Promise<void>',
+    },
+    {
+      name: 'didDismiss',
+      description: 'Emitted after the modal has dismissed.',
+      signature: 'Promise<void>',
+    },
+  ];
+
+  methods: ApiDescriptionMethod[] = [
+    {
+      name: 'scrollToTop',
+      description:
+        'Scrolls to the top of the modal body. It takes an optional input of "KirbyAnimation.Duration", fx. KirbyAnimation.Duration.SHORT. This method will have no effect if "scrollDisabled" is set to "true"',
+      signature: 'void',
+    },
+    {
+      name: 'scrollToBottom',
+      description:
+        'Scrolls to the bottom of the modal body. It takes an optional input of "KirbyAnimation.Duration", fx. KirbyAnimation.Duration.SHORT. This method will have no effect if "scrollDisabled" is set to "true"',
+      signature: 'void',
+    },
+  ];
+
+  footerProperties: ApiDescriptionProperty[] = [
+    {
+      name: 'type',
+      description:
+        'Sets the type of the footer. When inline the footer will have a transparent background and no shadow.',
+      type: ['fixed', 'inline'],
+      defaultValue: 'fixed',
+    },
+  ];
+
+  controllerProperties: ApiDescriptionProperty[] = [
+    {
+      name: 'flavor',
+      description: `(Optional) The flavor of the modal. 
+      
+      Modals with \`modal\` flavor have a close button placed in the top right corner and are full-screen on small screens.`,
+      defaultValue: 'modal',
+      type: ['modal'],
+    },
+    {
+      name: 'component',
+      description: 'The component which will be rendered inside the modal.',
+      defaultValue: '',
+      type: ['Component'],
+    },
+    {
+      name: 'componentProps',
+      description: '(Optional) The data to pass to the modal component.',
+      defaultValue: 'undefined',
+      type: ['undefined | { [key: string]: any; }'],
+    },
+    {
+      name: 'cssClass',
+      description: `(Optional) Adds custom css classes to the modal. This allows for custom styling of the modal (see 'CSS Custom Properties' section).`,
+      defaultValue: 'undefined',
+      type: ['string | string[]'],
+    },
+    {
+      name: 'canDismiss',
+      description: `(Optional) If canDismiss is true, then the modal will close when users attempt to dismiss the modal. If canDismiss is false, then the modal will not close when users attempt to dismiss the modal.
+      
+      canDismiss can also be a function, which must return a Promise that resolves to either true or false. If the promise resolves to true, then the modal will dismiss. If the promise resolves to false, then the modal will not dismiss.
+      `,
+      defaultValue: 'undefined',
+      type: ['boolean | (() => Promise<boolean>)'],
+    },
+    {
+      name: 'backdropDismiss',
+      description: `(Optional) If true, the modal will be dismissed when the backdrop is clicked.`,
+      defaultValue: 'true',
+      type: ['boolean'],
+    },
+    {
+      name: 'showBackdrop',
```

Comment:
Yes it still possible, because even if `showBackdrop` is false, then it is rendered in the DOM with a transparent background. I have added that detail to the description of `backdropDismiss `.